src/gs-details-page.c

```diff
@@ -2334,6 +2559,22 @@ gs_details_page_setup (GsPage *page,
 	self->builder = g_object_ref (builder);
 	self->cancellable = g_object_ref (cancellable);
 
+	self->plugin_status = GS_PLUGIN_STATUS_FINISHED;
+
+	self->volume_monitor = g_volume_monitor_get ();
+	self->removable_mounts = NULL;
+	g_signal_connect (self->volume_monitor, "mount-added",
+			  G_CALLBACK (gs_details_page_mount_added_cb),
+			  self);
+	g_signal_connect (self->volume_monitor, "mount-removed",
+			  G_CALLBACK (gs_details_page_mount_removed_cb),
+			  self);
+	mounts = g_volume_monitor_get_mounts (self->volume_monitor);
+	for (l = mounts; l; l = l->next) {
+		gs_details_page_mount_added_cb (self->volume_monitor, l->data,
+						self);
+	}
```

Comment:
I also think this is too much responsibility for the details view. Usually views in g-s shouldn't have so much feature-related logic. Instead, this should be in a plugin, which would affect the apps' properties accordingly (like we do for the shortcuts). The EOS plugin also already tracks mountpoints, so this logic should be there, and maybe we just add a `COPIABLE` prop to the app (or do it using the app's metadata if easier), so the details views (and others maybe) will be notified if an app is copiable or not.